tests/clickhouse/testdata/migrations/00001_a.sql

```diff
@@ -0,0 +1,55 @@
+-- +goose Up
```

Comment:
Will add a few more migrations to test for the various ClickHouse migration types to get a good spread.